doc/old/base_element.rst

```diff
@@ -24,13 +24,15 @@ via their tags, as shown below.
   dictionary instead of the keys normally yielded by iterating a :class:`dict`.
 
 A :class:`~dataset.Dataset` could be created directly, but you will
-usually get one by reading an existing DICOM file::
+usually get one by reading an existing DICOM dataset from file::
 
   >>> import pydicom
   >>> from pydicom.data import get_testdata_file
```

Comment:
I wonder if it makes sense to also move this to `examples`, maybe with another name (something like `data_path`) and reduced functionality, and deprecate `dta.get_testdata_file`. I would prefer to not have that in the documentation at all. Or are there use cases outside of examples (like reading user provided data stores)?

Reply:
I thought about something like that, I would've liked to be able to do something like 
```python
from pydicom import examples

path = examples.ct.path
```
But I didn't want to confuse users about expecting `FileDataset` having a `path` attribute. Another option was to just reference `examples.ct.filename`, which I guess is fine, if a bit odd (retrieve the dataset to get a path to use that path to `dcmread` to get the dataset). And as you've mentioned we could do something like
```python
from pydicom import examples

path = examples.get_path(examples.ct)  # Also a bit odd
path = examples.data_path("ct")  # ?
```

Reply:
Hm, yes - I guess `examples.ct.filename` would be ok, but I also don't like it. I thought about something like `examples.data.ct` and `examples.names.ct` instead, though I'm not sure if that is feasible and really better.